### ISSUE TITLE
English docs perhaps?

### ISSUE THREAD

**Issue by suesunss** [hugegraph/hugegraph#142]
### Expected behavior 期望表现
{type something here...}

It's natural to compare JanusGraph and HugeGraph. 

When I surf over JanusGraph community (groups), people seems to have much interests about HugeGraph. But, the documentation seems to be a barrier to them. 

I suggest HugeGraph communty to put some efforts into welcoming more contributors all around the world. We could start, for example, from translating the docs into English. 

Best,

### Actual behavior 实际表现
{type something here...}

Only English docs provided.

**Comment by javeme** [comment]
@suesunss Yes, this is a good idea. We also hope that everyone can participate in this work.

**Comment by suesunss** [comment]
@javeme Sure, very glad to be part of it.

**Comment by carlspring** [comment]
Hi guys,

It's great that you've put so much efforts into this project and that you've been developing it so actively!

However, the online (and international) community has adopted English as its main language, as it's more commonly used across the globe and more people are likely to understand it among developers. I appreciate that the project is being developed mainly by Chinese-speaking people, but perhaps it would be more successful and beneficial for the rest of the world, if you get more documentation translated to English.

It's also peculiar that all of the Github issue/pull templates have English section titles, but then everything else is in Chinese.

Any plans on translating your docs and perhaps switching to English? :)

Kind regards,

Martin

**Comment by javeme** [comment]
We plan to complete this task in 2022, as HugeGraph has joined the Apache Foundation Incubation. 
There is a lot of work and some help may be needed, if anyone would like to contribute, we are very welcome and grateful.

**Comment by imbajin** [comment]
Done in: https://github.com/apache/hugegraph-doc/tree/master/content/en
